lib/scaleway/charts/q-ingress-tls/templates/ingress-qovery.j2.yaml

```diff
@@ -0,0 +1,60 @@
+{%- if routes|length >= 1  %}
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ sanitized_name }}
+  namespace: {{ namespace }}
+  labels:
+    ownerId: {{ owner_id }}
+    routerName: {{ sanitized_name }}
+    routerId: {{ id }}
+    envId: {{ environment_id }}
+    fqdn: "{{ router_default_domain }}"
+  annotations:
+    external-dns.alpha.kubernetes.io/hostname: {{ router_default_domain }}
+    external-dns.alpha.kubernetes.io/ttl: "300"
+    external-dns.alpha.kubernetes.io/target: "{{ external_ingress_hostname_default }}"
```

Comment:
You can remove the target, and let external dns use the address set by the ingress controller. It will allow automatic update if the address change